plugins/lastfm.py

```diff
@@ -87,7 +88,12 @@ def api_request(method, **params):
     params.update({"method": method, "api_key": api_key})
     request = requests.get(api_url, params=params)
 
-    data = request.json()
+    try:
+        data = request.json()
+    except JSONDecodeError:
+        request.raise_for_status()
```

Comment:
Its unclear other than the name that this can move execution out of this block before the `raise`--add a comment specifying it or wrap it in its own try/except or similar

Reply:
Fixed in 84dac4fa089888ba7cf635073cdbc3041125a291